helpers/nodeHelpers.js

```diff
@@ -0,0 +1,16 @@
+
+function checkValidWriteInput(voltage, node) {
+    if(typeof(voltage) === "number" && (voltage <= 10 && voltage >= 0) ){
+        return true 
+    }
```

Comment:
volatage type is number? not int, float etc? can you define a constant for upper and lower limit